### ISSUE TITLE
[iOs] Bug on compilation XCode 11.4 (Swift 4 compiler)

### ISSUE THREAD

**Issue by massimodeluisa** [rdlabo/capacitor-facebook-login#24]
## General info
Platofrm: **iOs**
XCode version: **11.4**
Swift compiler used: **4.2**
Plugin version: **1.5.0**

## Bug info
Good morning, there's a bug at line _38_ of the `Plugin.swift` (@rdlabo/capacitor-facebook-login/ios/Plugin/Plugin/Plugin.swift) file:

```swift
                    call.reject("LoginManager.logIn failed", error)
```

which produces a string conversion error that prevents the build od the package.

## Possible solution
To solve that problem I simply removed the error from the `reject` method and recompiled the project.

@rdlabo can you please fix this?
Thank you in advance 😄!

**Comment by rdlabo** [comment]
@massimodeluisa massimodeluisa Capacitor 1.5.0 can't use in Swift 11.4, and It may take some time until Capacitor 2 is released.
You can downgrade Xcode to 11.3.
https://forum.ionicframework.com/t/xcode-11-4/185770/3

Thanks.

**Comment by danielrf121** [comment]
Running with "@capacitor/ios": "^2.0.0-beta.1", 

..same error. Will the correction be made only in the version 2 release?

**Comment by danielrf121** [comment]
For those who are in a hurry to distribute the app and want a paleative solution, comment the line:

**call.reject ("LoginManager.logIn failed", error)**

as the @massimodeluisa said above.

**Comment by rdlabo** [comment]
Capacitor 2.0.0 released, and I release version 2.0.x
https://github.com/rdlabo/capacitor-facebook-login/commit/a7c80c69adc8a282ca589b81c023f9fd111b9ed9

Please update latest. Thanks.
